internal/cri/server/images/image_remove.go

```diff
@@ -34,17 +34,24 @@ import (
 // Remove the whole image no matter the it's image id or reference. This is the
 // semantic defined in CRI now.
 func (c *GRPCCRIImageService) RemoveImage(ctx context.Context, r *runtime.RemoveImageRequest) (*runtime.RemoveImageResponse, error) {
+	resp, err := c.CRIImageService.RemoveImage(ctx, r.Image)
+	if err != nil {
```

Comment:
should probably just return resp, err for now since there should never be a case where there is a returned err and the resp is not err.. same for if there is a response there should never be an error...

Reply:
this is mainly because we return success on not found..

Reply:
yeah that is the reason I retained it. Are you suggesting we want to return error on not found now?